src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ConfigSet.cs

```diff
@@ -436,8 +436,6 @@ internal static AdamInstance FindAnyAdamInstance(DirectoryContext context)
 
             try
             {
-                string entryName = (string)rootEntry.Properties["distinguishedName"].Value;
```

Comment:
This doesn't have any side effects that would necessitate it be kept?

Reply:
This one only does a List.Copy on this getter. So i would say we have no side effects

Reply:
DirectoryEntry.Properties is just a derived Hashtable.